src/bar/BarChartWithFilters.vue

```diff
@@ -38,14 +38,19 @@
                                    @input="changeFilter(filter.id, $event)"></filter-select>
                 </div>
             </div>
-            <div v-if="!!xAxisLabel" id="chart" class="col-md-9">
+            <div v-if="!!xAxisLabel" id="chart" class="col-md-9" style="position:relative;">
                 <bar-chart-with-errors
                         :chart-data="processedOutputData"
                         :xLabel="xAxisLabel"
                         :yLabel="indicatorLabel"
                         :yFormat="formatValueFunction"
                         :show-errors="showRangesInTooltips"
                         style="width: 100%; height: 100%;"></bar-chart-with-errors>
+                <div v-if="showNoDataMessage" id="noDataMessage" class="px-3 py-2" style="width: 60%; position: absolute; border: 1px solid #adb5bd; bottom: 50%; left: 25%; background: #fff; margin: auto;">
```

Comment:
Classes added to the repo's stylesheet do not seem to get picked up hence the in line styling.

Reply:
They should do I think - could you either have a look into that, or should at least be able to put style into a <style> section in this file?